plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowDependencyManager.java

```diff
@@ -126,25 +126,23 @@ public boolean hasTrackedDataflowDependency(IProject project) {
   }
 
   public VersionRange getDataflowVersionRange(IProject project) {
-    Dependency dfDependency =
-        getDataflowDependencyFromModel(getModelFromProject(project));
-    if (dfDependency != null) {
-      String version = dfDependency.getVersion();
-      if (Strings.isNullOrEmpty(version)) {
-        return allVersions();
-      } else {
-        try {
-          return VersionRange.createFromVersionSpec(version);
-        } catch (InvalidVersionSpecificationException e) {
-          throw new IllegalStateException(
-              String.format(
-                  "Could not create version range from existing version %s", version),
-              e);
+    Model model = getModelFromProject(project);
+    if (model != null) {
+      Dependency dependency = getDataflowDependencyFromModel(model);
+      if (dependency != null) {
+        String version = dependency.getVersion();
+        if (!Strings.isNullOrEmpty(version)) {
+          try {
+            return VersionRange.createFromVersionSpec(version);
+          } catch (InvalidVersionSpecificationException e) {
+            String message =
+                String.format("Could not create version range from existing version %s", version);
```

Comment:
seems like overkill for format when a trivial concatenation would work equally well